README.md

```diff
@@ -1,4 +1,8 @@
-<img width="50%" alt="Box Elements" src="https://repository-images.githubusercontent.com/95743138/c161b500-021b-11ea-8bf9-3aa8776acdec" />
+<p align="center">
+  <img width="50%" alt="Box Elements" src="https://repository-images.githubusercontent.com/95743138/c161b500-021b-11ea-8bf9-3aa8776acdec" />
+  <img width="50%" src="https://github.com/box/sdks/blob/master/images/box-dev-logo.png" alt= “box-dev-logo”>
```

Comment:
I think since `alt` attributes are used by screen readers, the text should be more human readable:
```
alt="Box Developer logo"
```
\* not entirely sure if we should do `Box Dev` or `Box Developer` though
\** the double quotes used for the `alt` attribute look different than the other double quotes. we should make them the same